star-rx72n-firmware/lib/rx_motor/src/rx_motor.c

```
@@ -47,13 +47,16 @@ typedef enum {
 
 /** @brief Motor configuration validation limits (NASA Rule 5 compliance) */
-typedef enum {
+typedef enum : uint32_t {
   k_motor_min_pwm_freq  = 1000,  /**< Minimum PWM frequency (1 kHz) */
   k_motor_max_pwm_freq  = 50000, /**< Maximum PWM frequency (50 kHz) */
   k_motor_min_dead_time = 100,   /**< Minimum dead-time (100 ns) */
   k_motor_max_dead_time = 10000, /**< Maximum dead-time (10 us) */
 } motor_validation_limits_t;
 
+/* Zero duty for stopped outputs (floats can't be enums). */
+static const float s_duty_zero = 0.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)